src/main/java/uk/gov/hmcts/reform/sendletter/tasks/MarkLettersPostedTask.java

```diff
@@ -70,25 +73,25 @@ public void run(LocalTime now) {
         }
     }
 
-    private void updatePrintedAt(LetterPrintStatus letterPrintStatus) {
+    private void markAsPosted(LetterPrintStatus letterPrintStatus) {
         Optional<Letter> optional = repo.findById(letterPrintStatus.id);
         if (optional.isPresent()) {
             Letter letter = optional.get();
             if (letter.getStatus() == LetterStatus.Uploaded) {
                 letter.setPrintedAt(Timestamp.from(letterPrintStatus.printedAt.toInstant()));
                 letter.setStatus(LetterStatus.Posted);
                 repo.save(letter);
-                logger.info("Marking letter {} as {}", letter.getId(), LetterStatus.Posted);
+                logger.info("Marked letter {} as posted", letter.getId());
```

Comment:
should we not use enum? not too fuss about it

Reply:
if we're not marking it as posted we should rename the class, method name and everything else :)